phpunit

```diff
@@ -41,7 +41,9 @@ if (!file_exists("$PHPUNIT_DIR/phpunit-$PHPUNIT_VERSION/phpunit")) {
     $zip->close();
     chdir("phpunit-$PHPUNIT_VERSION");
     passthru("$COMPOSER remove --no-update symfony/yaml");
+    passthru("$COMPOSER require --dev --no-update symfony/phpunit-bridge '>=2.8@dev'");
     passthru("$COMPOSER install --prefer-source --no-progress --ansi");
+    copy('../../src/Symfony/Bridge/PhpUnit/SkippedTestsListener.php', './vendor/symfony/phpunit-bridge/SkippedTestsListener.php');
```

Comment:
This line should be removed before merging. For now, it allows testing before subtree-splitting